spdx-utils/build.gradle

```diff
@@ -86,16 +86,55 @@ task generateSpdxLicenseList {
 
         def enumValues = spdxLicenseIds.collect { "    ${licenseToEnumEntry(it.toString())}" }.sort().join(',\n') + ';'
         licenseEnumFile.append(enumValues)
-        licenseEnumFile.append("""
+        licenseEnumFile.append('''
 
                 companion object {
                     /**
                      * Return the enum value for the given [id], or null if it is no SPDX license id.
                      */
                     fun forId(id: String) = SpdxLicense.values().find { it.id == id }
                 }
+
+                /**
+                 * The full license text as a string.
+                 */
+                val text = javaClass.getResource("/licenses/$id").readText()
```

Comment:
Consider using `by lazy { ... }` so that not all license texts are read into memory when the class is loaded.

Reply:
Good idea, will do.